guides/extensions/redux-store/index.md

```diff
@@ -0,0 +1,5 @@
+# Aries Framework JavaScript Redux Store
+
+Aries Framework JavaScript Redux Store is a wrapper around [Aries Framework JavaScript](https://github.com/hyperledger/aries-framework-javascript.git) (AFJ), built with [Redux Toolkit](https://redux-toolkit.js.org/). It provides a implementation of state management that can be used to build React & React Native SSI clients using AFJ.
```

Comment:
This readme is inconsistent with the other packages. As they're all very minimal, can yiu make them consistent?